packages/verifiable-credentials/test/create-credentials.test.ts

```diff
@@ -83,6 +85,14 @@ describe('Credential tests', function () {
 
             assert.deepEqual(vc.issuanceDate, issuanceDate, 'issuanceDate options matches generated VC date ');
         });
+        it('Ensure issuanceDate generated in VC is within 10secs', async () => {
+            const jwt: any = await credential.createCredentialJWT(config.SUBJECT_DID, config.CREDENTIAL_DATA);
+            const decodedCredential = await credential.verifyCredential(jwt.didJwtVc)
+            const payload = decodedCredential.payload
+            const vc = payload.vc
+
+            assert.ok(vc.issuanceDate, 'issuanceDate is within 10secs after creating ');
```

Comment:
This isn't testing issuanceDate is created within 10 seconds of creation date....

Reply:
@tahpot,
Please, I think I would need help here, how do I implement this, how do I compare the time, its unclear to me .

Reply:
You need to compare the `issuanceDate` is within 10 seconds of the current time.

The idea is that the verifiable credentials library automatically set the `issuanceDate` and we want to test it was created at the current time. It doesn't need to be 10 seconds, it could be 5.

Reply:
This would happen only when the `issuanceDate` is not provided right?

Reply:
Yes, correct.